learn/tanssi/proxy-accounts.md

```diff
@@ -0,0 +1,202 @@
+|  Max proxies   | {{ networks.dancebox.proxy.max_proxies }} proxies  |
+
+## Proxy Types {: #proxy-types }
+
+When creating a proxy account, you must choose a type of proxy that will define how the proxy can be used. The available options are:
+
+- **`Any`** - allows the proxy account to use any function supported by the proxy pallet. There is no filtering of calls.
+- **`CancelProxy`** - allows the proxy account to reject and remove any announced proxy calls
+- **`Staking`** - allows the proxy account to perform staking-related transactions, such as block producer and `authorMapping()` functions 
+- **`NonTransfer`** - this type of proxy account is allowed to submit any type of transaction with the exception of balance transfers
+- **`Balances`** - allows the proxy account to only make transactions related to sending funds
+- **`Registrar`** - allows the proxy account to only make transactions related to the registrar pallet
+- **`SudoRegistrar`** - allows the proxy account to only make transactions related to the registrar pallet that need to be called by Sudo
+
+For this guide, you will be setting up a proxy account using the balances proxy type. Since this type enables the proxy to spend funds on behalf of the primary account, you should exercise caution and only provide access to accounts you trust. The proxy will have access to transfer all of the funds within the primary account, and if not trusted, the proxy could drain the primary account. Ensure that you maintain oversight of your proxy accounts and remove any proxies that are no longer needed.
+
+## Creating a Proxy Account {: #creating-a-proxy-account }
+
+There are a couple of ways you can create proxy accounts in [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/accounts){target=\_blank}, either from the **Extrinsics** page or the **Accounts** page. However, to create a time-delayed proxy, you will need to use the **Extrinsics** page. A time delay provides an additional layer of security to proxies by specifying a delay period based on the number of blocks. This will prevent the proxy account from executing a transaction until the delay period ends. The delay allows time for the primary account that controls the proxy to review pending transactions, potentially for malicious actions, and cancel if necessary before execution.
+
+You also have the option of creating a proxy of type **Any** which grants the proxy account full and unrestricted control over the primary account. This means that the proxy account can transfer funds, and perform any arbitrary action. The following demo will showcase configuring a **Balances** proxy, which is more restrictive than a proxy of type **Any**, but allows for transferring funds, which makes it perfect for demonstration purposes. After configuring your **Balances** proxy you can try transferring funds from the primary account via proxy. 
```

Comment:
```suggestion
The following demo will showcase configuring a Balances proxy, which allows for transferring funds, making it perfect for demonstration purposes. After configuring your Balances proxy you can try transferring funds from the primary account via proxy. 
```

Some of this seems out of place here, as the previous section defines what an Any proxy type does